pom.xml

```diff
@@ -30,7 +30,7 @@
   </licenses>
 
   <scm>
-    <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+    <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
```

Comment:
`git` protocol has been removed from GitHub, there were warnings in the maven build saying this would be removed in the future.